### ISSUE TITLE
Not working with Python3?

### ISSUE THREAD

**Issue by howels** [OutsourcedGuru/OctoPrint-USBControl#11]
After upgrading to Python3  (ironically to get another plugin to work) it seems that USBControl is not visible/compatible.

Does this work with Python3?

**Comment by arndl** [comment]
I have the same problem: USBControl is not shown under settings. 
Script has to be "sudo ~/oprint/lib/python3.7/site-packages/usbcontrol/bin/install". But where to hit "save"??

**Comment by ErikZandboer** [comment]
Same here. The plugin is not listed in Octoprint "get more", after manually adding it through the URL provided I do see the plugin sitting there, but when I activate it there is a "thunderflash" icon next to it which says when highlighted "usbcontrol incompatible to octoprint or the underlying python envrionment". Using Octoprint v1.5.3.

![image](https://user-images.githubusercontent.com/16650934/111343020-4b752580-867b-11eb-8de3-f6dbb16f33b5.png)

**Comment by JHale716** [comment]
Looks like the plugin needs updating, Python >=2.7,<3 says it only works with the older version of python... You'll need to give the owner of the plugin a nudge about updating it to Pyton 3.

**Comment by ErikZandboer** [comment]
Yes pretty sure that’s the issue. Have no clue on how complex that would be. I remember from one of my own projects that a tool like 2to3 can sometimes fix it for 99% already,
I was just struggling with the way strings were handled I think.
